src/main/java/net/mcreator/workspace/Workspace.java

```diff
@@ -408,7 +408,7 @@ public static Workspace readFromFS(File workspaceFile, @Nullable Window ui)
 							"Unsupported generator", JOptionPane.WARNING_MESSAGE);
 					GeneratorConfiguration generatorConfiguration = GeneratorSelector.getGeneratorSelector(ui,
 							GeneratorConfiguration.getRecommendedGeneratorForFlavor(Generator.GENERATOR_CACHE.values(),
-									currentFlavor), currentFlavor, false);
+									currentFlavor), currentFlavor, null);

```

Comment:
Here it was false before, so it is not new workspace, thus here it should not be null, right?

Please test changes again